packages/transformers/js/src/JSTransformer.js

```diff
@@ -104,7 +105,7 @@ export default (new Transformer({
 
         if (!ignore) {
           traverse.cache.clearScope();
-          traverse(ast.program, fsVisitor, null, {asset, logger, ast});
+          traverse(ast.program, fsVisitor, null, {asset, logger, ast, options});
```

Comment:
Should we skip this whole traversal unless `unsafeInlining` is true?

Reply:
We still need `process.env.NODE_ENV` and `process.browser`